README.md

```diff
@@ -49,96 +30,98 @@ Via Bower:
 
 The adal.js source is [here](https://github.com/AzureAD/azure-activedirectory-library-for-js/tree/master/lib/adal.js).
 The adal-angular.js source is [here](https://github.com/AzureAD/azure-activedirectory-library-for-js/tree/master/lib/adal-angular.js).
-## Samples, tests and documentation
 
-For a sample demonstrating basic usage of ADAL JS please refer to [this repo](https://github.com/AzureADSamples/SinglePageApp-DotNet).
+## Build and run tests
```

Comment:
Personally, I would move "Build and run tests" after "Usage". As a user I want to understand how easy this will be to use it, because I even think of contributing

Reply:
I agree, will update.